build_defs/internal_do_not_use/j2cl_application.bzl

```diff
@@ -31,9 +35,14 @@ def j2cl_application(
         name: name of the rule.
         entry_points: JavaScript namespace of the entry point for the app.
         deps: dependencies of the app (e.g. j2cl_library, js_library, etc).
+        extra_defs: Extra flags to pass to the Closure Compiler.
+        formatting: Whether to pretty-print or minify the code.
         rewrite_polyfills: rewrite ES6 library calls to use polyfills provided
           by the compiler's runtime to support "old" browsers. Only affects
           production binary.
+        language: Output target language to pass to Closure Compiler. If no
+          value is passed, the default is used (`ECMASCRIPT5` at the time of
+          this writing).
```

Comment:
`language`: ability to change the output target away from the default (`ECMASCRIPT5`, i believe)